CHANGELOG.md

```diff
@@ -56,8 +56,9 @@ The table below shows which release corresponds to each branch, and what date th
 | [3.0.0](#300)    |          | Aug 20, 2016
 | [2.2.0](#220)    |          | Jan 5, 2015
 
-## 4.5.0 (`dev`)
 
+## 4.5.0 (`dev`)
+- [#1722][1722] Added Crypto class and cypher suites
```

Comment:
Please try to keep these sorted. not change the blank lines formatting, and add a link as the other entries have below